### PR TITLE
Jm updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This PR includes the following updates:

- Breaking change consisting of renaming models to be consistent with current Fivetran best practices
- Custom schema compatibility
- CircleCi integration
- Github Pages docs creation
- Updates which address Issue #5 and Issue #3 
- Version upgrade from v0.2.1 -> v0.3.0 as this release will include breaking changes
- Documentation updates

### CODE REVIEW

File: models/stripe__invoice_line_items.sql [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    invoice.due_date,
+    invoice.amount_due,
+    invoice.subtotal,
+    invoice.tax,
+    invoice.total,
+    invoice.amount_paid,
+    invoice.amount_remaining,
+    invoice.attempt_count,
+    invoice.description as invoice_memo,
+    invoice_line_item.unique_id  as invoice_line_item_id,
```
Comment (fivetran-joemarkiewicz) [inline models/stripe__invoice_line_items.sql:52]: This update was to reflect bug noticed by Issue #3

File: models/stripe__invoice_line_items.sql [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    invoice_line_item.description as line_item_desc,
+    invoice_line_item.amount as line_item_amount,
+    invoice_line_item.quantity,
+    invoice_line_item.period_start,
+    invoice_line_item.period_end,
```
Comment (fivetran-joemarkiewicz) [inline models/stripe__invoice_line_items.sql:57]: This update was to reflect enhancement noted in Issue #5

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          name: "Run Tests - BigQuery"
+          environment:
+              GCLOUD_SERVICE_KEY_PATH: "/home/circleci/gcloud-service-key.json"
+
+          command: |
+            . venv/bin/activate
+            echo `pwd`
+            cd integration_tests
+            dbt deps
+            dbt seed --target bigquery
```
Comment (fivetran-kristin) [inline .circleci/config.yml:54]:
```suggestion
            dbt seed --target bigquery --full-refresh
```

File: dbt_project.yml [hunk 4]
```diff
@@ -1,12 +1,39 @@
...
 
 name: 'stripe'
-version: '0.0.1'
+version: '0.3.0'
 
 require-dbt-version: [">=0.18.0", "<0.20.0"]
 
 models:
     stripe:
+        +schema: _stripe
```
Comment (fivetran-kristin) [inline dbt_project.yml:10]:
```suggestion
        +schema: stripe
```

**Review by fivetran-kristin — APPROVED** [review]
Minor change suggestions, otherwise looks good!

File: .circleci/config.yml (hunk 3 above)
Comment (fivetran-joemarkiewicz) [reply .circleci/config.yml]: Updated

File: dbt_project.yml (hunk 4 above)
Comment (fivetran-joemarkiewicz) [reply dbt_project.yml]: Updated

**Pull request merged**